tutorials/36_Building_Pipelines_With_Conditional_Routing.ipynb

```diff
@@ -0,0 +1,540 @@
+{
```

Comment:
<p>How about having the title be "Building Fallbacks to Websearch with Conditional Routing"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/287/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Great, thanks for the idea!